Makes second set of changes to IPI docs (#113832)
openshift/openshift-docs: modules/installation-azure-arm-tested-machine-types.adoc

[id="installation-azure-arm-tested-machine-types_{context}"]
= Tested instance types for Azure on 64-bit ARM infrastructures

[role="_abstract"]

> You can use this reference to find Microsoft Azure ARM64 instance types tested with {product-title}. Choose a listed instance type when you install a cluster on 64-bit ARM infrastructure.

Reviewer: Is there a way to write this where it's not self referential? Maybe:

```suggestion
There are several Microsoft Azure ARM64 instance types tested with {product-title}. Choose a listed instance type when you install a cluster on 64-bit ARM infrastructure.
```
